src/Operation/Factory/SubresourceOperationFactory.php

```diff
@@ -142,13 +142,19 @@ private function computeSubresourceOperations(string $resourceClass, array &$tre
                 if (!\in_array($rootShortname, $operation['shortNames'], true)) {
                     $operation['shortNames'][] = $rootShortname;
                 }
+
```

Comment:
Please add `stateless` to `ROUTE_OPTIONS` and revert all other changes in this file.